src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/Utils.groovy

```diff
@@ -127,6 +134,26 @@ public class Utils {
     }
 
     /**
+     * Finds the parameterized type argument for a {@link MethodsToList} class and returns it.
+     *
+     * @param c A class.
+     * @return The parameterized type argument for the class, if it's a {@link MethodsToList} class. Null otherwise.
+     */
+    @Whitelisted
+    public static Class<Describable> getMethodsToListType(Class c) {
+        Class retClass
+        c.genericInterfaces.each { Type t ->
+            if (t instanceof ParameterizedType) {
+                if (t.rawType.equals(MethodsToList.class) && t.getActualTypeArguments().first() instanceof Class) {
```

Comment:
Is there a way to have type without ActualTypeArguments? first() may throw NPE in such case IIRC